client/components/summary/README.md

```diff
@@ -28,6 +28,9 @@ render: function() {
 
 * `label` (required): A string description of this value, ex "Revenue", or "New Customers"
 * `value` (required): A string or number value to display - a string is allowed so we can accept currency formatting.
+* `href` (required): An internal link to the report focused on this number.
```

Comment:
Minor: The README example should include `href` since it is a required prop.

Should it also mention that '/analytics' is default, from defaultProps, Or is that just temporary for these examples?

Reply:
I intended for the /analytics default to be temporary, but yeah the readme example should get updated.